tasks/javascript/easy/sum.js

```diff
@@ -2,3 +2,7 @@
 
 // TODO: Write a function that returns the sum of two numbers
 
+const sum = (a, b) => {
+    return a + b;
+}
```

Comment:
This works, but if you are using arrow functions, it is recommended to omit the curly brackets and return-keyword since they are not needed in arrow functions.

So you can just write:
```bash
const sum = (a, b) => a + b
```

Reply:
Thanks... I really forgot, I'm used to writing it like that, but thanks for the reminder.